delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/StorageFileRejection.scala

```diff
@@ -138,6 +137,9 @@ object StorageFileRejection {
           s"File cannot be saved because it already exists on path '$path'."
         )
 
+    final case class FileNotFound(path: String)
```

Comment:
I don't know if we can rename it in a way that makes it different from the `FileNotFound` from `FileRejection` ?